tests/test_cache_manager.py

```diff
@@ -176,6 +177,8 @@ def test_eviction_when_too_big(cm_client, tmpdir):
     assert cm.lru == {str(file1): 5, str(file2): 4}
     assert cm.total_size == 9
 
+    time.sleep(0.5)
```

Comment:
I started hitting problems with a couple of cache manager tests where I think they're running too fast so the access time sometimes isn't different to create the situation we're asserting against.

This adds some minor sleeps to ensure the test situation is created.